sdk/openai/openai/samples/v1-beta/javascript/streamCompletionsOnly.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Sorry I wasn't clear, my previous comment about completionsRest.ts also applies to this one.